src/unistd.rs

```diff
@@ -1754,6 +1754,40 @@ pub fn mkstemp<P: ?Sized + NixPath>(template: &P) -> Result<(RawFd, PathBuf)> {
     Ok((fd, PathBuf::from(pathname)))
 }
 
+/// Creates a directory which persists even after process termination
+///
+/// * `template`: a path whose 6 rightmost characters must be X, e.g. `/tmp/tmpdir_XXXXXX`
+/// * returns: filename
+///
+/// Err is returned either if no temporary filename could be created or the template doesn't
+/// end with XXXXXX
+///
+/// See also [mkstemp(2)](http://pubs.opengroup.org/onlinepubs/9699919799/functions/mkdtemp.html)
+///
+/// ```rust
+/// use nix::unistd;
+///
+/// match unistd::mkdtemp("/tmp/tempdir_XXXXXX") {
+///     Ok(_path) => {
+///         // do something with directory
+///     }
+///     Err(e) => panic!("mkdtemp failed: {}", e)
+/// };
+/// ```
+#[inline]
```

Comment:
You should remove the `#[inline]`

Reply:
I did the same as for the mkstemp function